pkg/daemon/daemon.go

```diff
@@ -22,6 +22,7 @@ import (
 	mcfgclientv1 "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
 	"github.com/pkg/errors"
 	"github.com/vincent-petithory/dataurl"
+	fsnotify "gopkg.in/fsnotify.v1"
```

Comment:
Minor/optional: this is from `gofmt` I suppose? We usually try to split those out as separate commits to make review easier.